ES/ESmy021.xml

```diff
@@ -247,4 +341,4 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <ab/>
       </body>
    </text>
-</TEI>
```

Comment:
add `<change>` element (in both files - and all the others)

Reply:
ok